hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/SqlPlanImpl.java

```diff
@@ -1055,6 +1055,7 @@ static class SelectPlan extends SqlPlanImpl {
         private final SqlRowMetadata rowMetadata;
         private final PlanExecutor planExecutor;
         private final List<Permission> permissions;
+        private final Map<String, Map<String, Expression<?>>> partitionStrategyCandidates;
```

Comment:
worth documenting meaning of key and value on both levels of Map

Reply:
still not documented here

Reply:
https://github.com/hazelcast/hazelcast/pull/24813/commits/96b0110c977e907bf2d85495319989801588cfd2